### ISSUE TITLE
cors support

### ISSUE THREAD

**Issue by harsh-tamr** [nshou/elasticsearch-kibana#19]
would be nice to have cors unblocked

**Comment by nshou** [comment]
I'm sorry @harsh-tamr I didn't notice this ticket. If you are still having any issue, please let me know.

**Comment by txj-xyz** [comment]
CORS is not enabled by default, if you're having any more issues since the latest update feel free to open this again.
